content/blog/2018-12-11-treestartr.md

```diff
@@ -0,0 +1,101 @@
+
+Starting trees are often gleaned from the literature. Usually, a previously published phylogeny will be input as a starting point. In my research we combine fossil and molecular data, and we usually don't have a published phylogeny for a set of taxa. I can't simply pluck a tree from the internet and pop it into my analysis. I need a reproducible way to automate the process of generating starting trees. I am faculty at a primarily undergraduate institution, meaning many of my employees are just gaining their skills as computational biologists. Therefore, I need a reasonable way to get starting trees that is simple enough to be executed and debugged by undergraduate student who might not have their legs all the way under them. Tall order!
+
+This is where [`treestartr`](https://ropensci.github.io/treeStartR/) came from. 
+
+From the user side, treestartr has four main utilities: to add taxa to a tree at random, to add taxa to a tree by guessing at taxonomy, to add taxa by hand, or to add taxa via a spreadsheet. All of these facilitate different aspects of the research, and I'll talk about each in the context of how my students are using them. 
+
+### Adding tips via dataframe
+
+I'm presently working on a dated phylogeny with ants. Two of my lab members, Christina Kolbmann and Tyler Tran, are interested in fossil taxon sampling and its relationship to divergence time estimation accuracy and precision. Ants have tons of fossils, including multiple fossils in some genera or subfamilies. This is great - it means we have lots of data. But not everyone else does. This wealth of data allows us to ask big picture questions about what happens if we  exclude parts of the data. One question we're interested in asking is if subsampling fossils is has deleterious effects on accuracy and precision. 
+
+In these cases, we know roughly where on the tree might be a reasonable starting place to try attaching the taxon. In this case, we can use the function [`text_placr`](https://ropensci.github.io/treeStartR/#adding-tips-via-csv) to read in a dataframe of taxonomic relationships. We can then use those relationships to generate a starting tree. An example taxonomy dataframe is below. In this data frame the two tips to be added, _Kretzoiartcos beatrix_ and _Ursus abstrusus_, are listed in the `taxon` column. The `clade` column contains tips that are currently on the tree. The two tips associated with _Kretzoiartcos beatrix_, _Indarctos arctoides_ and _Indarctos vireti_, will form a monophyletic group with _Kretzoiartcos beatrix_ on the generated starting tree. The three taxa associated with _Ursus abstrusus_ will form a monophyletic group with _Ursus abstrusus_. 
+
```

Comment:
Any chance you could add a figure/ sketch of the tree you are describing in the text so that readers can picture "tree", "tips", "monophyletic group"? I'm familiar with the terms but still had trouble picturing it from the data frame and text without getting distracted from the flow of the post.